### DISCUSSION TITLE
Backend as a service

### DISCUSSION THREAD

**Discussion by poboisvert** [cloudbet/docs#18] [Ideas]
I was wondering if I could have an API KEY or access to use cloudbet API as a backend. Augur Turbo is offering a solution similar. My goal is to used CloudBet like https://thoughtmachine.net/ for fintech

**Comment by kgravenreuth** [comment]
Hi @poboisvert ,
We can definitely give you API key access to use the Cloudbet API. The process is explained in more detail in https://github.com/Cloudbet/docs/discussions/16

As a summary:

If you have not done so yet, please register your Cloudbet account on our [Sign Up Page](https://www.cloudbet.com/en/auth/sign-up?utm_source=github.com&utm_medium=community&utm_campaign=axis) . You can then deposit the equivalent of 10 EUR in a currency of your choice and obtain your API Key from the [Account -> API Page](https://www.cloudbet.com/en/player/api?utm_source=github.com&utm_medium=community&utm_campaign=axis) (available while logged in).

You can use the Cloudbet API to fetch odds as well as place bets on sporting events offered by Cloudbet. Here are some resources to get you started:

1. [Cloudbet Feed API Docs](https://www.cloudbet.com/api/?utm_source=github.com&utm_medium=community&utm_campaign=axis)
2. [Cloudbet Betting API Docs](https://www.cloudbet.com/api/?urls.primaryName=Trading&utm_source=github.com&utm_medium=community&utm_campaign=axis)
3. [Detailed Cloudbet Betting API Tutorial](https://www.cloudbet.com/en/blog/posts/sports-betting-api-tutorial-with-golang?utm_source=github.com&utm_medium=community&utm_campaign=axis) on the Cloudbet blog which guides through account creation, obtaining a Cloudbet API Key and then fetching odds + placing bets.
4. [Cloudbet NPM SDK and market helper package](https://www.npmjs.com/package/@cloudbet/market-helper). There is also a [CodeSandbox Demo for Cloudbet API](https://codesandbox.io/s/cloudbet-odds-api-example-we1vk?file=/src/App.js) which demonstrates how to render odds from the Cloudbet API.
5. [Postman Collection for Cloudbet Feed, Trading and Account APIs](https://www.postman.com/cloudbet/workspace/cloudbet-api/overview)

Hope that's helpful. Please raise any other questions you have here and we'll get them clarified right away.

**Comment by kgravenreuth** [comment]
Hi @poboisvert 

Could you please reach out to us via email to affiliates@cloudbet.com for more detailed discussions? Thanks.
